drivers/serial/uart_nrfx_uarte.c

```diff
@@ -28,6 +28,7 @@ struct uarte_nrfx_data {
 	uart_irq_callback_user_data_t cb; /**< Callback function pointer */
 	void *cb_data; /**< Callback function arg */
 	u8_t *tx_buffer;
+	volatile bool disable_tx_irq;
```

Comment:
I think the type should be a atomic_t type.

Reply:
Only independent load and store operations are used which are itself atomic, there is no need for atomic type.